www/src/components/layer-model/layer-model.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react"
+import hex2rgba from "hex2rgba"
+
+import { space, colors, radii } from "../../utils/presets"
+import LayerIcon from "../../assets/icons/layer-icon"
+import {
+  ContentLayerContent,
+  BuildLayerContent,
+  DataLayerContent,
+  ViewLayerContent,
+  AppLayerContent,
+} from "./layer-content-sections"
+
+const Layer = ({ layer, onClick, selected, children }) => {
+  const { baseColor, title } = layer
+
+  return (
+    <button
+      onClick={onClick}
```

Comment:
I love how these are focusable and they work with the keyboard. This component looks like a tab switcher to me, though, so it would be amazing to make it function with one tab stop for the controls. Here's the ARIA Authoring Practices Guide for tabpanels, which will show semantics and navigation conventions: https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel

And here's a Codepen demo I made that might help: https://codepen.io/marcysutton/pen/oKBzdr

Reply:
I think I got this all sorted out! Maybe I should have just used reach UI's tabs but I think I've got everything the spec requires 🙂